DependencyInjection/Configuration.php

```diff
@@ -72,8 +71,15 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
-            ->end()
-        ;
+                ->arrayNode('toolbar')
+                    ->canBeDisabled()
```

Comment:
regarding kernel.debug: could we make enabled a simple field and default it to the value kernel.debug? then people could just explicitly enable/disable it if they explicitly enabled the toolbar despite kernel.debug being false. if kernel.debug is not available here, we could have a choice of `true`, `false` and `"auto"`, with auto meaning the DI extension classs looks at kernel.debug.